src/Normalizer/AdherentNormalizer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace AppBundle\Normalizer;
+
+use AppBundle\Entity\IdeasWorkshop\Vote;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+final class AdherentNormalizer implements DenormalizerInterface
+{
+    private $normalizer;
+    private $tokenStorage;
+
+    public function __construct(NormalizerInterface $normalizer, TokenStorageInterface $tokenStorage)
+    {
+        if (!$normalizer instanceof DenormalizerInterface) {
```

Comment:
why do you not typehint with this interface?

Reply:
Done.